### ISSUE TITLE
[a11y]: Link - JAWS is not announcing the link state as "Visited" for the visited link

### ISSUE THREAD

**Issue by 2nikhiltom** [carbon-design-system/carbon#16396]
### Package

@carbon/react

### Browser

Firefox

### Package version

v1.56.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

When the `visited` property in the playground is set to "true," JAWS doesn't announce the link as a "Visited" link, but the visual color change indicating its visited state.

### Recommend Fix
Ensure JAWS read the state of the link as "visited"

### Steps to reproduce

https://react.carbondesignsystem.com/?path=/story/components-link--playground&args=visited:!true;size:lg

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Since this playground example doesn’t exist in our Storybook (@carbon/react v1.84), we’re not able to reproduce or verify the issue as described. I believe we can close this issue.
